tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -483,7 +406,7 @@
     spec:
       nodeSets:
       - openstack
-      {%+ if edpm_nodes_networker is defined +%}
+      {%+ if edpm_nodes_networker is defined or edpm_networker_deploy +%}
```

Comment:
I think we should have this in `defaults/main.yaml` defaulted to false?

Reply:
added